experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -234,14 +236,14 @@ def _patch_label_2d(
                 path_prefix = f"{outdir}/{batch_indexes[i][0]}_{batch_indexes[i][1]}"
                 model_output = model_output.detach().cpu()
                 # save image:
-                image_to_disk(np.array(batch[i, 0, :, :]), path_prefix + "_img.png")
+                image_to_disk(np.array(batch[i, 0, :, :]), path_prefix + "_img.png", float(img.min()), float(img.max()))
```

Comment:
Is it possible to use the max and min values here from the config file?

Reply:
do you mean now, or in the next attempt to address the "global MIN and MAX" next sprint? Currently we don't have MIN and MAX defined in the cofing, I reverted those changes!

Reply:
okay that makes sense.. sorry if I misunderstood.